BSP/STM32F413/Makefile

```diff
@@ -239,13 +239,13 @@ $(BUILD_DIR):
 .INTERMEDIATE: build_msg
 
 c_compile_msg:
-	@echo -e "${GREEN}Compiling C files...${NC}"
```

Comment:
why are you getting rid of the -e option here? It's needed for escape characters to work